recipes/resolvelib/LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2018, Tzu-ping Chung <uranusjr@gmail.com>
```

Comment:
The `LICENSE` file is included in the PyPI tarball, so you shouldn't include that here.

Reply:
Removed it, thanks!